record/record.go

```diff
@@ -25,6 +26,12 @@ type LineageRecord struct {
 	Type string `json:"type"`
 }
 
+type Owner struct {
+	URN  string `json:"urn"`
```

Comment:
I'm thinking if we should have an email field too 🤔 .
There might be a case where URN won't be an email. It could be an ID or even username e.g. `john.doe`.

wdyt? @mabdh @ravisuhag

Reply:
For now, would it be a problem if `Name` could contain `e-mail` or `Full name` or any specific identifier of the owner?
The different between `urn` and `name` should be just `name` could be non-unique while `urn` should be unique across our platform context.

Reply:
Putting email in the name could be misleading. If we want to differentiate then email is better I would say.

Reply:
Make sense, I will update assets schema and add a new field called email in this `Owner` struct. I will also update the `Owner` in meteor sink. (Although in the future we could just use columbus package to do that.

@StewartJingga in that case we need to update the extractor too.

Reply:
Created

https://github.com/odpf/proton/pull/80

https://github.com/odpf/meteor/issues/283